.github/ISSUE_TEMPLATE/REQUEST_A_FEATURE.yml

```
@@ -0,0 +1,20 @@
+name: '🚀 Request a feature'
+description: 'Do you want to suggest a new feature?'
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to request a feature!
+  - type: textarea
+    id: what-problem
+    attributes:
+      label: 'What is the problem you're trying to solve?'
```

Fixing YAML syntax error: 👇🏼 

```suggestion
      label: "What is the problem you're trying to solve?"
```